File: JaveScript-Mocha/test/test_rose_garden.js
var {expect} = require('chai');
var {Shop, Item} = require('../src/rose_garden.js');
describe("Rose Garden", function() {

  it("should foo", function() {
    const roseGarden = new Shop([ new Item("foo", 0, 0) ]);
    const items = roseGarden.updateQuality();
    expect(items[0].name).to.equal("fixme");
  });

});

File: python/test_rose_garden.py
# -*- coding: utf-8 -*-
import unittest

from rose_garden import Item, RoseGarden


class RoseGardenTest(unittest.TestCase):
    def test_foo(self):
        items = [Item("foo", 0, 0)]
        rose_garden = RoseGarden(items)
        rose_garden.update_quality()
        self.assertEquals("fixme", items[0].name)

        
if __name__ == '__main__':
    unittest.main()
